templates/base/markup_codepreview.tmpl

```diff
@@ -0,0 +1,21 @@
+<div class="code-preview-container file-content">
+	<div class="code-preview-header">
+		<a href="{{.FullURL}}" class="muted" rel="nofollow">{{.FilePath}}</a>
+		Lines {{.LineStart}} to {{.LineStop}} in
```

Comment:
Localize?

Reply:
It could be like this 8f7e57acdacc52e680115809007a241d1c068576

I didn't do it because I don't want to cause unnecessary arguments about using HTML in translations.

Reply:
Looks good like that. Yes we should forbid new HTML in translations.